scripts/compress.js

```diff
@@ -5,6 +5,7 @@ const source = process.argv[2]
 const destination = process.argv[3]
 
 if (!source || !destination) {
+  // eslint-disable-next-line
```

Comment:
In this case more reasonable disable concrete rule for whole file 
`/* eslint-disable no-console */`
At first line of file